tests/container

```diff
@@ -233,6 +234,36 @@ else
     lxc exec c1 -- grep -F "delegate_cmds=map_create:prog_load,delegate_maps=ringbuf,delegate_progs=socket_filter,delegate_attachs=cgroup_inet_ingress" /proc/self/mountinfo
 fi
 
+echo "==> Test running miscellaneous binary formats with binfmt_misc in unprivileged container"
+if lxc info | grep -F 'unpriv_binfmt: "true"'; then
+	lxc start c1
+	waitInstanceBooted c1
+
+	echo "Enabling binfmt_misc in unprivileged container"
+	lxc exec c1 -- mount binfmt_misc -t binfmt_misc /proc/sys/fs/binfmt_misc
+    [ "$(lxc exec c1 -- cat /proc/sys/fs/binfmt_misc/status)" = "enabled" ]
+
+	echo "Installing required packages and compiling aarch64 test binary"
+	lxc exec c1 -- apt-get update
+	lxc exec c1 -- apt-get install --no-install-recommends -y qemu-user-static gcc-aarch64-linux-gnu binfmt-support libc6-dev-arm64-cross
+	lxc exec c1 -- systemctl restart systemd-binfmt # Restart systemd-binfmt service to pick up new binary types provided by qemu-user
+	lxc exec c1 -- test -f /proc/sys/fs/binfmt_misc/qemu-aarch64
```

Comment:
This has more to do with verification so I'd put it right before `./hello == "Hello, World!"`